src/loglist.cpp

```diff
@@ -311,6 +317,21 @@ void qtLogCallback(
   }
 }
 
+void logToStdout(bool b)

```

Comment:
Does this log to stdout only or causes it to log to UI + log file + stdout? (+ whatever... )

Reply:
Logs everywhere. There's actually already a sink set up in uibase for the console, it's just never useful because MO is not a console application, so stdout doesn't go anywhere. All this does is connect stdout to the parent process' console, or create one.